openpathsampling/high_level/transition.py

```diff
@@ -257,10 +257,18 @@ def ensemble_statistics(self, ensemble, samples, weights=None, force=False):
         hist_data = {}
         buflen = -1
         sample_buf = []
+        prev_sample = {h: None for h in run_it}
+        prev_result = {h: None for h in run_it}
         for sample in in_ens_samples:
             for hist in run_it:
                 hist_info = self.ensemble_histogram_info[hist]
```

Comment:
True -- but I'll add that to #666. You might as well just merge that one, which will auto-merge this as well.